integration_test/src/python/integration_test/core/integration_test_bolt.py

```diff
@@ -98,6 +98,7 @@ def process(self, tup):
 
   def emit(self, tup, stream=Stream.DEFAULT_STREAM_ID, anchors=None,
            direct_task=None, need_task_ids=False):
+    Log.info("emitting tuple: %s", tup)
```

Comment:
pl remove

Reply:
I intentionally left this comment in here so it is easier to debug the integration tests.  Tuples received is already printed but tuples emitted was missing